make.go

```diff
@@ -537,48 +537,35 @@ func normalizeDebianProgramName(str string) string {
 }
 
 func shortHostName(gopkg string, allowUnknownHoster bool) (host string, err error) {
+
+	knownHosts := map[string]string{
+		"github.com":        "github",
+		"code.google.com":   "googlecode",
+		"cloud.google.com":  "googlecloud",
+		"gopkg.in":          "gopkg",
+		"golang.org":        "golang",
+		"google.golang.org": "google",
+		"gitlab.com":        "gitlab",
+		"bitbucket.org":     "bitbucket",
+		"bazil.org":         "bazil",
+		"blitiri.com.ar":    "blitiri",
+		"pault.ag":          "pault",
+		"howett.net":        "howett",
+		"go4.org":           "go4",
+		"salsa.debian.org":  "debian",
```

Comment:
Would it make sense to also take the opportunity to alphabetize this list to help improve future maintainability?

Reply:
since it has been merged, I will update it in another PR.

Reply:
sort it in #125